.gitignore

```diff
@@ -31,3 +33,6 @@ __pycache__
 /mingw
 /spidermonkey
 /binaryen
+
+/scripts/cpython
+/scripts/python-*
```

Comment:
In general I try to resist adding stuff to gitignore.    In this case I kind of like that I can cleanup the build stuff with `git clean`.   If you disagree then maybe we can make a single `build` or `out` directory that we can ignore and have the scripts use that do to their building.

Reply:
I think we should have in .gitignore the files that the build scripts in this very repository itself generate, that will be most straightforward. What is the downside in having these in here anyway?

Reply:
`git clean -x` is the standard flag to clean also in .gitignore. It sounds like what you want to run is a `git clean -xdf` to nuke everything including gitignore?

Reply:
Sounds like it is better to leave that for later, I don't have time to spend more back and forth on this but need to land this support. The .gitignore contents are not relevant to M1 support, so left this out.